share/tools/gmt_aliases.csh

```diff
@@ -16,7 +16,7 @@ if ($? == 1) then
   exit 1
 endif
 
-set gmt_modules = (`gmt --show-classic`)
+set gmt_modules = ($(gmt --show-classic))
```

Comment:
Now all tests pass for me. 

Please revert the changes to `gmt_aliases.csh`, then this PR is good to merge.